liquidpromptrc-dist

```diff
@@ -1,240 +0,0 @@
-
```

Comment:
I agree that this file should be deleted, but there are many places that it is explicitly referenced:
```bash
$ grep -RniI liquidpromptrc-dist
docs/config.rst:29:Liquid Prompt ships with an example config file, ``liquidpromptrc-dist``. You
docs/config.rst:32:    cp ~/liquidprompt/liquidpromptrc-dist ~/.config/liquidpromptrc
liquidprompt:626:            # shellcheck source=liquidpromptrc-dist
contrib/dist/archlinux-aur/PKGBUILD:37:  install -Dm644 liquidpromptrc-dist "$pkgdir/etc/liquidpromptrc"
contrib/dist/debian/create_deb.sh:8:cp ../../liquidpromptrc-dist ./liquidprompt/etc/liquidpromptrc
```
These ideally would all be replaced with references to the new script, or in the case of the scripts in `contrib/dist/`, replaced with calls to the script.

The tricky one is going to be in `liquidprompt` itself, as we won't have a file laying around for `shellcheck` to use. Probably the simplest and best solution is to replace the line with `# shellcheck source=/dev/null`.